app-crypt/gpgme/gpgme-1.9.0.recipe

```diff
@@ -29,19 +29,22 @@ PROVIDES="
 	cmd:gpgme_tool${secondaryArchSuffix} = $portVersion compat >= 1.5
 	lib:libgpgme${secondaryArchSuffix} = 11.18.0 compat >= 11
 	lib:libgpgmepp${secondaryArchSuffix} = 6.4.0 compat >= 6
+	lib:libqgpgme${secondaryArchSuffix} = 7.2.0 compat >= 7
```

Comment:
These versions should be converted to the latest format (and as expected, someone forgot to bump this since 1.5-1.6). Also, remove braces when unneeded.

```diff
 ARCHITECTURES="!x86_gcc2 x86 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="11.18.0"
+libCPlusVersion="6.4.0"
+libQtVersion="7.2.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libCPlusVersionCompat="$libCPlusVersion compat >= ${libCPlusVersion%%.*}"
+libQtVersionCompat="$libQtVersion compat >= ${libQtVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%.*}"
+
 #TODO: fix gpgme-config hardcoded package paths
 PROVIDES="
- 	gpgme${secondaryArchSuffix} = $portVersion compat >= 1.5
-	cmd:gpgme_tool${secondaryArchSuffix} = $portVersion compat >= 1.5
-	lib:libgpgme${secondaryArchSuffix} = 11.18.0 compat >= 11
-	lib:libgpgmepp${secondaryArchSuffix} = 6.4.0 compat >= 6
-	lib:libqgpgme${secondaryArchSuffix} = 7.2.0 compat >= 7
+ 	gpgme$secondaryArchSuffix = $portVersionCompat
+	cmd:gpgme_tool$secondaryArchSuffix = $portVersionCompat
+	lib:libgpgme$secondaryArchSuffix = $libVersionCompat
+	lib:libgpgmepp$secondaryArchSuffix = $libVersionCPlusCompat
+	lib:libqgpgme$secondaryArchSuffix = $libVersionQtCompat